Gerg12/voting-system
# Add Enhanced Styling, Results Persistence, and Testing Support

**Changes Made:**

**Enhanced Styling:**
- Implemented a more visually appealing design with box shadows and border-radius for tables and buttons.
- Styled office buttons with a better color scheme and added hover effects.
- Applied consistent spacing and font adjustments for better readability.

**Results Persistence:**
- Modified the app to save user votes and results to local storage.
- Created a results.js file to store and retrieve voting results.
- Improved the handling of voting results, ensuring they persist across page reloads.

**Testing Support:**
- Introduced Jest and React Testing Library for automated testing.
- Wrote initial tests for the UserList component, covering rendering and user interaction scenarios.
- Configured Jest and set up a test environment to support ongoing testing efforts.

**ResultsList Component:**
- Refactored the app structure to include a ResultsList component for displaying voting results.
- Separated the rendering logic for results into the new component for better organization.

**Miscellaneous:**
- Cleaned up unused files and code snippets.
- Ensured code consistency and adherence to best practices.
- Added explanatory comments for better code understanding.

**Testing Instructions:**
- Run `npm run test` to execute the automated tests for the UserList component.
- Manually test the application to verify the enhanced styling and results persistence features.
